jupyterbook/content/ioos_installation_conda.md

```diff
@@ -29,9 +29,9 @@ you might choose `c:\programs\Miniconda3`.
 On the "Advanced Installation Options" screen,
 leave the boxes checked if you want Miniconda 3 to be your default python.
 If you are going to be switching from Python 2 to Python 3 or perhaps some other Python distribution,
-it's best uncheck the boxes and use the `Anaconda Command Prompt` or `Anaconda Navigator` (see below for instructions) to start Anaconda.
+it is best uncheck the boxes and use the `Anaconda Command Prompt` or `Anaconda Navigator` (see below for instructions) to start Anaconda.
 
-### Linux/OS X
+### Linux/macOS
 
 Copy-and-paste this:
```

Comment:
should indicate that this should be ran in the terminal.

Reply:
Good catch! We do have that below but not here.